core/src/actions/settings.ts

```diff
@@ -29,6 +30,28 @@ export class SettingsList extends AuthenticatedAction {
   }
 }
 
+export class SettingClusterNameView extends OptionallyAuthenticatedAction {
```

Comment:
Doesn't this information also come back from the navigation action?  We do display the cluster name in the UI when you are logged out already.  The navigation action is already pre-loaded and available on every UI page though `props.navigation`

Reply:
Sure does. I adjusted the structure of it to get a little more information (to know if it has been set or not). But we're only using it in the nav, so it was a quick fix.